index.html

```diff
@@ -4,15 +4,44 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz-App</title>
   </head>
   <body>
-    <nav>
-      <a href="./index.html">Homepage</a>
-      <a href="./bookmark.html">Bookmark</a>
-      <a href="./input.html">Input</a>
-      <a href="./profile.html">Profile</a>
-    </nav>
-    <p>Homepage</p>
+    <header>
+      <h1>Quiz-App</h1>
+    </header>
+    <main>
+      <article>
+        <div class="bookmark"></div>
```

Comment:
instead of an div class just for the bookmark picture, you can use `<img>` because its an inline element and not a block element like a div. then you can use position: absolute to put in on the right place :)